hydra-main/src/main/java/com/addthis/hydra/minion/Minion.java

```diff
@@ -259,8 +262,12 @@ private Minion(@JsonProperty("dataDir") File rootDir,
         }
         activeTaskKeys = new HashSet<>();
         jetty = new Server(webPort);
+
+        PrometheusServletCreator.create(handler);
+        handler.start();
```

Comment:
Is it normal to start and stop handlers outside of Jetty? I don't see pattern this in any other code changed in this PR.

Reply:
handler start moved to inside of jetty.